src/index.js

```diff
@@ -107,6 +107,40 @@ export default class S3Database implements ILocalData {
     return new S3PackageManager(this.config, packageName, this.logger);
   }
 
+  async search(onPackage: Function, onEnd: Function, validateName: Function) {
```

Comment:
Conflict ahead ... https://github.com/Remitly/verdaccio-s3-storage/pull/18 🤓

Reply:
ha...well should be a pretty easy one to fix either way 😄